### DISCUSSION TITLE
Left-hand panel, please make width adjustable

### DISCUSSION THREAD

**Discussion by ThatAdamGuy** [athensresearch/athens#1162] [Feature Requests]
I really appreciate that the right-hand panel width is adjustable.
Please also do the same for the _left-hand_ panel.
Thank you!

**Comment by jefftangx** [comment]
This features makes sense!

But what is the problem you are trying to solve with an adjustable left sidebar?

Would supporting multiple widths (e.g. normal, wide, extra wide) also address your problem?

**Reply by ThatAdamGuy** [reply]
Thanks for checking!

For me, the right-hand panel has way way too much empty space.  It means that I end up having to have my overall Athens window much larger than I'd like.

But if there were options like: Compact, Normal, and Wider, that'd be fine with me :)
